tests/deploy/api/test_e2e.py

```diff
@@ -86,6 +90,21 @@ def test_e2e(tmp_local_root, multiple_answer_chunks, stream_answer, corpus_name)
         assert len(chat["messages"]) == 1
         assert chat["messages"][-1] == message
 
+        corpuses = client.get("/corpuses").raise_for_status().json()
+        assert corpuses == {source_storage: [corpus_name]}
+
+        corpuses = (
+            client.get(f"/corpuses?source_storage={source_storage}")
```

Comment:
Let the library handle the URL building.

```suggestion
            client.get("/corpuses", params={"source_storage": source_storage})
```

Same for the occurrence below.